_layouts/wiki.html

```diff
@@ -15,15 +15,6 @@
     <div id="maincontainer" class="fx-col-100-xs fx-col-50-m">
       <noscript class="red"><strong>{{ site.data.general.tNoJSEnabled }}</strong></noscript>
       <main lang="{{ page.lang }}">{{ content }}</main>
-      <div id="editsitecontent">
-        <a href="{{ site.githubrepoeditbase }}/{{ page.path }}" target="_blank" rel="noopener noreferrer">
-          <svg aria-labelledby="etitle" width="20" height="20" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg">
-            <title id="etitle">{{ site.data.general.btnEditAlt }}</title>
```

Comment:
Sorry, I must have missed that in #574: You should also remove all the btnEditAlt entries in _data/*/general.yml
That's my fault.

Reply:
Ok I will remove them too.

Reply:
Done

Reply:
Thanks!